src/data/emerson.json

```diff
@@ -687,6 +687,69 @@
           "description": "Used for link hover colour for light profile components on contrasting surface backgrounds"
         }
       }
+    },
+    "shadow": {
+      "base": {
+        "value": "{colors.shades.black}",
```

Comment:
Don't believe we have an official colour set for Emerson yet. Just a placeholder value